### DISCUSSION TITLE
Supabase UI, consideration to switch the Icon library

### DISCUSSION THREAD

**Discussion by ericfennis** [supabase/supabase#3175] [Feature Requests]
I'm maintainer and co-creator of the [Lucide](https://github.com/lucide-icons/lucide) Icon library a community fork of Feather icons. We started a fork one year ago because the feather icon library repo is no longer actively maintained.
Sinds then we almost hitting 500+ of icons. And we have now 40 contributors added new icons to this library.

I noticed this new awesome tool and the love for Open Source software. I saw you using Feather Icons for the Supabase UI. 
I want to introduce you to [Lucide](https://github.com/lucide-icons/lucide) same style as feather but more extended and maintained icon library by old Feather community members.

Maybe a nice replacement candidate for the current used icon library in @superbase/ui

**Comment by prescience-data** [comment]
Not directly involved with Supabase, but that's great to know that Feather isn't actively maintained anymore - I use it in a lot of projects so will make the switch. Thanks!

**Comment by MildTomato** [comment]
I'd love to hear any recommendations for icon libraries.

I'd also be keen to get Supabase UI to work with any icon library as well, although that might be a bit too ambitious.

@ericfennis this is great work! this looks fantastic, we should consider switching to this.
are the icon names still the same for those that you inherited from Feather?

**Reply by ericfennis** [reply]
@MildTomato Yeah I understand, awesome!
Yess for 98% we still uses the same names. We renamed some (1-3 times) because the names were not logical with the icon it represents.
But if you use the react library and simply change `react-feather` to `lucide-react` you will see it in your terminal if some are not found.

**Reply by MildTomato** [reply]
nice! it probably makes sense to switch. In future, it would be great that we support a few different icon packs but this looks like a great suggestion and we get more icons to play with in the immediate short term 👍 💯

**Reply by MildTomato** [reply]
I'll look into this later - I'd rather make sure we are not tied into an icon pack to closely (like we are now) but rather make sure that anyone can just use whatever icon pack they prefer.

**Comment by MildTomato** [comment]
We have started using [Lucide](https://github.com/lucide-icons/lucide) Icon library in the Studio 🚀 

https://github.com/supabase/supabase/blob/f475b16321c6cc9805056254d48b83c0243248fe/packages/ui/package.json#L65
